src/main/interface/settings.c

```diff
@@ -526,6 +526,7 @@ const clivalue_t valueTable[] = {
 #endif
 #ifdef USE_DUAL_GYRO
     { "gyro_to_use",                VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_GYRO }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_to_use) },
+    { "gyro_for_axis[3]",              VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_GYRO }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_for_axis[3]) }
```

Comment:
see https://github.com/betaflight/betaflight/blob/aa7b1a2f6f35a3f31404b9f073448c5cc4dca089/src/main/interface/settings.c#L546, it uses array to handle XYZ

Or create 3 variables, `gyro_for_axis_x`, `gyro_for_axis_y`, `gyro_for_axis_z`, using addresses into gyro_for_axis array